packages/stores/src/queries-external/ibc-status/types.ts

```diff
@@ -0,0 +1,18 @@
+export interface IbcMetrics {
+  data: {
+    source: string;
+    destination: string;
+    channel_id: string;
+    token_symbol: string;
+    token_name: string;
+    last_tx: string;
+    size_queue: number;
+    duration_minutes: number;
+  }[];
+}
+
+export enum IbcStatus {
```

Comment:
With TypeScript, you can define a `const enum` that will map to 0-indexed integers at compile time.

Reply:
Hmmm weird I am getting this error when setting the enum to const. 

`Type error: Cannot access ambient const enums when the '--isolatedModules' flag is provided.`